app/src/sharedTest/java/org/oppia/app/topic/practice/TopicPracticeFragmentTest.kt

```diff
@@ -6,16 +6,19 @@ import android.content.res.Configuration
 import androidx.test.core.app.ActivityScenario
 import androidx.test.core.app.ApplicationProvider
 import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
```

Comment:
Remove these unused imports in this file.

Reply:
Done